related-files-file.el

```diff
@@ -0,0 +1,58 @@
+;;; related-files-file.el --- Provide methods and jumpers for files  -*- lexical-binding: t; -*-
```

Comment:
"Add support for file-based places to related-files"?

Reply:
Fixed in 6f2f029